Lecture-4/Task2/Task2/Application.java

```diff
@@ -0,0 +1,37 @@
+package Task2;
+
+import java.util.Scanner;
+
+public class Application {
+
+    public static void main(String[] args) {
+        System.out.println("type the day number: ");
+        Scanner scanner = new Scanner(System.in);
+        int dayNumber = scanner.nextInt();
+        switch (dayNumber){
+            case 1:
+                System.out.println("Monday");
```

Comment:
You should extrac duplicated code to private methods when possible:
`private static void printInfo(String info)`